packages/sdk/src/swap/sdk.ts

```diff
@@ -201,15 +201,14 @@ export class SwapSDK {
   }
 
   async getSwapLimits(): Promise<{
-    minimumSwapAmounts: ChainAssetMap<bigint>;
     maximumSwapAmounts: ChainAssetMap<bigint | null>;
     minimumDepositAmounts: ChainAssetMap<bigint>;
   }> {
     const {
-      swapping: { minimumSwapAmounts, maximumSwapAmounts },
+      swapping: { maximumSwapAmounts },
       ingressEgress: { minimumDepositAmounts },
     } = await this.getStateChainEnvironment();
 
-    return { minimumSwapAmounts, maximumSwapAmounts, minimumDepositAmounts };
+    return { maximumSwapAmounts, minimumDepositAmounts };
```

Comment:
```suggestion
    return { maximumSwapAmounts, minimumSwapAmounts: minimumDepositAmounts };
```

Reply:
this is just to retain backward compatibility?
If so, maybe breaking it is actually not a bad idea. It will force them to consider correct implementation

Reply:
If it were a general purpose SDK, I think we could keep the name, but for a swapping SDK, this name is more fitting. I don't have too strong an opinion, but it should probably match the name of the field on the tokens

Reply:
agree with this - changed